src/microns_to_nwb/tools/nwb_helpers.py

```diff
@@ -15,7 +15,8 @@ def check_module(nwb, name, description=None):
 def start_nwb(scan_key):
     nwb = NWBFile(
         identifier=str(uuid4()),
-        session_description="unknown",
+        session_description="Contains calcium imaging recorded from multiple cortical visual areas "
+                            "and behavioral measurements while a mouse viewed natural movies and parametric stimuli.",
```

Comment:
this might well as be the abstract from [the article](https://www.biorxiv.org/content/10.1101/2021.07.28.454025v2.full.pdf)?